Solutions/CohesitySecurity/Tests/Alert.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+import json
+
+
+class Alert:
+    def __init__(self, json_str):
+        data = json.loads(json_str)[0]
+        self.jobId = None
+        self.clusterId = data['clusterId']
+        self.clusterIncarnationId = None
+        for prop in data['propertyList']:
+            if prop['key'] == 'jobId':
+                self.jobId = prop['value']
+            elif prop['key'] == 'clusterIncarnationId':
+                self.clusterIncarnationId = prop['value']
+        self.protectionGroupId = f"{self.clusterId}:{self.clusterIncarnationId}:{self.jobId}"
```

Comment:
Please assert if jobId or clusterIncarnationId or clusterId is empty here after parsing from the data.

I want to understand what happens in the case where the propertyList doesn't have jobId or clusterIncarnationId? Will there be a run time fault? Or will be default initialized to empty string?

Reply:
done